.travis.yml

```diff
@@ -8,6 +8,9 @@ install:
 notifications:
   slack:
     rooms:
-      - undeadbaby:$SLACK_KEY#random
+    - undeadbaby:WDGNd6FP2Uu61jNcbMaHLSIv#random
```

Comment:
It was not working at all which is why I opted for a plain text code swap
as I couldn't find the documentation on how to correct it.

On Wed, 27 Jul 2016, 12:02 laynecsmall, <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/operationzombie/Embedded-devices/pull/2#discussion_r72360395>
> :
>
> > @@ -8,6 +8,9 @@ install:
> >  notifications:
> >    slack:
> >      rooms:
> > -      - undeadbaby:$SLACK_KEY#random
> > +    - undeadbaby:WDGNd6FP2Uu61jNcbMaHLSIv#random
>
> Is the key substitution not working correctly?
>
> keys in plain text stored in a repo is very bad form, even more so when
> the repository is public like this one.
> In this case it's not super serious, cause it just grants access to our
> slack which is hardly sensitive, but it'd be good to have key substitution
> working correctly in the case we do need to use something sensitive in the
> future.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/operationzombie/Embedded-devices/pull/2/files/e8360690be58cb5f7b920e5dcbda16e808fad5b8#r72360395>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ARK0l6hDzcxMVB4B5r1BuNsg4RPqe5mLks5qZqAXgaJpZM4JVr6v>
> .
>